OpenRA.Mods.Common/Traits/Render/WithProductionOverlay.cs

```diff
@@ -31,6 +31,8 @@ public class WithProductionOverlayInfo : ITraitInfo, Requires<RenderSpritesInfo>
 		[Desc("Custom palette is a player palette BaseName")]
 		public readonly bool IsPlayerPalette = false;
 
+		public readonly bool PauseOnLowPower = false;
```

Comment:
Or, even better, make this use upgrades now so that we don't need to worry about porting things back to upgrades later.